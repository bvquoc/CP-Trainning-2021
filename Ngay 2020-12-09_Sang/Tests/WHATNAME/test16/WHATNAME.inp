145 50
120 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
72 55 125 117 141 145 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 110 63 98 26 78 101 124 143 81 95 75 92 8 112 52 137 116 140 68
108 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 72 4 73 19 109 1 96 76 42 5 126 21
84 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68
108 130 87 115 127 133 139 104 40 10 60 23 50 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 8 112 52 137 116 140 68 55 125 117 141 145
96 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3
108 130 87 115 127 133 139 104 40 10 60 23 50 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
60 55 125 117 141 145 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68
96 130 87 115 127 133 139 104 40 10 60 23 50 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
108 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 22 66 89 136 144 24 7 64 77 135 103 83 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
96 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 17 71 46 132 107 20 59 53 131 39 29 84 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3
48 55 125 117 141 145 93 138 43 86 36 51 9 61 6 122 69 38 121 128 14 97 123 105 58 27 41 134 34 114 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68
96 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 72 4 73 19 109 1 96 76 42 5 126 21
84 55 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 125 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 145 117 141 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68
120 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
72 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 72 4 73 42 19 1 109 96 76 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 5 126 21 13 113 28 108 65 2 80 119 120 70 142 3
108 130 87 115 127 133 139 104 40 10 60 23 50 16 79 74 90 18 15 56 57 48 102 44 33 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145
84 130 87 115 127 133 139 104 40 10 60 23 50 42 5 126 21 16 79 74 90 18 15 56 57 48 102 44 33 4 72 73 19 1 109 96 76 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3
96 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
108 130 87 115 127 133 139 104 40 10 60 23 50 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
72 55 125 117 141 145 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 17 71 46 132 107 20 59 53 131 39 29 84 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 8 112 52 137 116 140 68
120 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
108 130 87 115 127 133 139 104 40 10 60 23 50 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 22 66 89 136 144 24 7 64 77 135 103 83 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
108 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
108 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 17 71 46 132 107 20 59 53 131 39 29 84 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
84 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 72 4 73 5 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 76 1 42 96 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 109 19 126 21
108 130 87 115 127 133 139 104 40 10 60 23 50 16 79 74 90 18 15 56 57 48 102 44 33 17 71 46 132 107 20 59 53 131 39 29 84 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
96 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 72 4 73 19 109 1 96 76 42 5 126 21
96 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3
84 55 125 117 141 145 72 4 73 19 109 1 96 76 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 5 126 21 22 66 89 136 144 24 7 64 77 135 103 83 42 8 112 52 137 116 140 68
60 72 4 73 19 109 1 96 76 42 5 126 21 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 22 66 89 136 144 24 7 64 77 135 103 83
108 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 72 4 73 19 109 1 96 76 42 5 126 21
60 55 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 145 125 17 71 46 132 107 20 59 53 131 39 29 84 117 121 128 14 97 123 105 58 27 41 134 34 114 141 8 112 52 137 116 140 68
108 130 87 115 127 133 139 104 40 10 60 23 50 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 12 85 62 45 35 30 106 47 49 118 82 67 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
96 93 138 43 86 36 51 9 61 6 122 69 38 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
108 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 121 128 14 97 123 105 58 27 41 134 34 114 110 63 98 26 78 101 124 143 81 95 75 92 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 8 112 52 137 116 140 68 55 125 117 141 145 72 4 73 19 109 1 96 76 42 5 126 21
60 72 4 73 19 109 1 96 76 93 138 43 86 36 51 9 61 6 122 69 38 16 79 74 90 18 15 56 57 48 102 44 33 91 25 31 88 37 111 11 54 94 100 129 32 42 5 126 21 22 66 89 136 144 24 7 64 77 135 103 83
96 130 87 115 127 133 139 104 40 10 60 23 50 93 138 43 86 36 51 9 61 6 122 69 38 91 25 31 88 37 111 11 54 94 100 129 32 17 71 46 132 107 20 59 53 131 39 29 84 12 85 62 45 35 30 106 47 49 118 82 67 22 66 89 136 144 24 7 64 77 135 103 83 13 113 28 108 65 2 80 119 120 70 142 3 72 4 73 19 109 1 96 76 42 5 126 21
12 130 91 22 121 93 13 17 110 8 16 12 72
12 87 4 25 128 66 138 71 113 63 79 112 85
12 52 115 31 14 43 89 46 98 28 74 73 62
12 45 108 127 88 137 97 86 19 132 26 136 90
12 35 133 65 37 123 116 36 107 109 78 18 144
12 30 24 139 111 2 105 51 140 20 101 1 15
12 106 104 7 11 58 80 9 96 59 68 124 56
12 57 55 47 40 76 54 27 64 61 53 119 143
12 48 49 125 10 94 42 41 6 77 131 81 120
12 102 70 118 60 117 100 134 5 122 39 135 95
12 103 44 141 82 142 23 126 129 34 69 29 75
12 92 21 33 67 83 50 32 3 114 38 145 84
