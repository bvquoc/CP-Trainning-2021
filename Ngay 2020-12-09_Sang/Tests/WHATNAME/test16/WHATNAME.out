44 44 49 40 48 47 45 39 45 47 45 39 39 41 44 39 39 43 42 44 49 39 49 44 40 42 46 41 49 44 41 49 49 49 43 43 43 49 48 46 47 47 41 49 42 41 46 47 47 49 44 41 46 46 46 45 46 45 45 48 46 41 40 46 43 40 49 45 49 48 40 39 41 41 49 46 47 43 40 45 47 49 49 49 40 42 40 42 41 42 39 49 39 47 48 45 42 41 3 48 44 48 49 45 44 45 43 42 43 39 44 40 40 49 41 43 48 48 46 47 39 48 43 45 47 49 42 40 49 39 47 42 43 48 48 42 42 40 44 44 49 49 46 43 49
